deployment/src/strongmind_deployment/container.py

```diff
@@ -43,6 +43,25 @@ def __init__(self, name, opts=None, **kwargs):
                                            tags=self.tags,
                                            opts=pulumi.ResourceOptions(parent=self),
                                            )
+
+        self.target_group = aws.lb.TargetGroup(
```

Comment:
is the right call here to let a redirect be a safe health check, or to include a better health check in our install? I think it's probably the latter.

Reply:
After discussion, actually the right call is to use rails' default health controller.